Wired_Map.conf

```
@@ -0,0 +1,14 @@
+team.2.color = blue
+team.2.pos = (2,-59,-58.5)
+team.1.pos = (2,-59,59.5)
+team.1.color = red
+team.1 = red
+author = olliy
+rotation = x
+h = 234
+team.2 = blue
+name = Wired Map 
+initial_stuff = default:pick_steel, default:sword_stone, defaut:torch 10, shooter:pistol
+r = 200
+license = CC BY SA 4.0
+screenshot = ![screenshot_20191002_102119](https://user-images.githubusercontent.com/55839810/66066472-7064ef80-e4fe-11e9-8fb3-d19453c7311d.png)
```

This needs to be changed to 
```suggestion
screenshot = Wired_Map.png
```
Also upload the screenshot and rename it to `Wired_Map.png`